assets/js/ViewerApp.tsx

```diff
@@ -67,10 +75,24 @@ class ViewerApp extends React.Component<
     });
     socket.connect();
 
-    const channel = socket.channel('signs:all', {});
-    channel.join().receive('ok', () => true);
+    const signsChannel = socket.channel('signs:all', {});
+    const headwaysChannel = socket.channel('headways:all', {});
+    const chelseaBridgeAnnouncementsChannel = socket.channel(
+      'chelseaBridgeAnnouncements:all',
+      {},
+    );
+    const signGroupsChannel = socket.channel('signGroups:all', {});
+    const alertsChannel = socket.channel('alerts:all', {});
+
+    [
+      signsChannel,
+      headwaysChannel,
+      chelseaBridgeAnnouncementsChannel,
+      signGroupsChannel,
+      alertsChannel,
+    ].forEach((channel) => channel.join().receive('ok', () => true));
```

Comment:
@losvedir do we want error handling when joining channels? I tried to reduce the repetitiveness here but I noticed while doing it that we don't have a `receive('error', resp => ...)`.

Reply:
Ah, I like the loop here. I'm not entirely sure what we'd _do_ in the event of an error. For that matter, it doesn't really look like we do anything, so we probably don't even need the `'ok'` callback. I'd say we can leave it like this for now; it hasn't been a problem for us these last years, as far as I can tell.